### DISCUSSION TITLE
Renewing of Domain

### DISCUSSION THREAD

**Discussion by vakadu11** [vercel/community#4420] [Help] [answered]
### Summary

Hi Everyone I'm having a hard time to look on the vercel platform on how to renew my vercel-bought domain. Also I have read the docs and according to them I should contact the support team for this. Thanks in advance.
![Screenshot 2023-10-09 at 5 25 22 PM](https://github.com/vercel/community/assets/74175232/9489b654-a500-41e9-9949-e1792c8414f8)

### Steps to Reproduce

N/A

**Comment by amyegan** [comment, marked as answer]
Hey @vakadu11! You can open a case at [vercel.com/help](https://www.vercel.com/help) with `Billing` as the problem area. That will create a ticket with the team that can help you sort out any billing problems and renew your domain 🙂

**Comment by nmn-administrator** [comment]
@amyegan I have the same issue(It would be great if a button was added to the drop-down list next to the domain name). I also tried to create a ticket, but I ran into another problem: after clicking the "Submit Case" button, the form remains filled in and no message (like "Your message has been sent") is shown. Does the form work at all?
